Core/Src/bmb.c

```diff
@@ -15,9 +15,12 @@
 /* ============================= DEFINES ============================== */
 /* ==================================================================== */
 #define BITS_IN_BYTE					8
+#define BITS_IN_INT						16
```

Comment:
`BITS_IN_INT` seems like a questionable name. I suppose this is true for the BMB. Maybe `BITS_IN_REGISTER`